tools/makeboards.py

```diff
@@ -298,7 +298,10 @@ def MakeBoard(name, chip, vendor_name, product_name, vid, pid, pwr, boarddefine,
         BuildFlashMenu(name, chip, 8*1024*1024, [0, 7*1024*1024, 4*1024*1024, 2*1024*1024])
         BuildFlashMenu(name, chip, 16*1024*1024, [0, 15*1024*1024, 14*1024*1024, 12*1024*1024, 8*1024*1024, 4*1024*1024, 2*1024*1024])
     elif name == "datanoisetv_picoadk_v2":
-        BuildPSRAM(name)
+            BuildPSRAM(name)
+            BuildPSRAMCS(name)
```

Comment:
`BukldPSRAMCS`, `BuildPSRAMFREQ`, and `BuildFreq` aren't needed here.  You have a hard-wired board and know the CS pin (which should be a `#define PSRAM_CS xxx` in `variants/datanoisetv_v2/pins_arduino.h`).  There's no reason to have these as a menu.  Only the first PSRAM size via `buildpsram` is needed here.  See the `challenger_2350_bconnect` or other RP2350 board w/PSRAM for more info...

Reply:
So I should just leave BuildPSRAM? Could you check the latest commit and maybe make necessary changes if possible? I am currently working on some example code.

Reply:
If your board always has PSRAM then there's no need for these menus.  Define the proper pin/clock/CS in the variant.h like in the other 2350+PSRAM boards and it will "just work."

If there are 2 boards, 1 with and 1 without, then only the BuildPSRAM menu makes sense to have because everything else if defined by the PCB.

`BuildFreq` is called 3 lines below, so the instance here is duplicate  That's not legal and I'm not sure what the IDE will do in that case.  Best case you end up with some weird dup's menu items.